neo/Prompt/Commands/Wallet.py

```diff
@@ -151,18 +152,13 @@ def execute(self, arguments):
             print("Wallet file not found")
             return
 
-        try:
-            passwd = prompt("[password]> ", is_password=True)
-        except KeyboardInterrupt:
-            print("Wallet opening cancelled")
-            return
```

Comment:
Why delete this?

Reply:
Good catch, added it back. I was probably trying out shutdown approaches and this might have been a test that failed with one of the approaches